pkg/controller/llmisvc/fixture/gwapi_helpers.go

```diff
@@ -33,7 +35,22 @@ const (
 	DefaultGatewayControllerName = "gateway.networking.k8s.io/gateway-controller"
 )
 
+func InCluster() bool {
+	_, err := os.Stat("/var/run/secrets/kubernetes.io/serviceaccount/token")
+	if err != nil {
+		log.Println("Not in cluster")
+	} else {
+		log.Println("In cluster, skipping")
+	}
+	return err == nil
+}
```

Comment:
I think we want to know if we are talking with actual cluster, not from within the cluster.

For that we can use:

```golang
if envTest.Environment.UseExistingCluster == ptr.To[bool](true) 
```